### PR TITLE
Partner Manager - Enhance topic on receive endpoint deployment

### PR DESCRIPTION
**Pull request by dhirsch1**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/deploy-onpremise.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ wrapper.java.maxmemory=4606
 ----
 +
 . xref:mule-runtime::installing-an-enterprise-license.adoc[Install an Enterprise License].
-. Add the server in Runtime Manager based on the instructions in xref:runtime-manager::servers-create#create-a-server[Add Servers to Runtime Manager].
+. Add the server in Runtime Manager based on the instructions in xref:runtime-manager::servers-create.adoc#create-a-server.adoc[Add Servers to Runtime Manager].
```
Comment (sathya0) [inline modules/ROOT/pages/deploy-onpremise.adoc]:
```suggestion
. Add the server in Runtime Manager based on the instructions in xref:runtime-manager::servers-create.adoc[Add Servers to Runtime Manager].
```

File: modules/ROOT/pages/deploying-receive-endpoints.adoc [hunk 2]
```diff
@@ -1,55 +1,69 @@
...
 
-You can deploy AS2 receive, HTTP receive, and HTTPS receive endpoints on a public or private port, or on-premises.
+You can deploy AS2 receive, HTTP receive, and HTTPS receive endpoints on a public port, private port, or on-premises.
 
 == Deploy to CloudHub on a Public Port
 
-When an AS2, HTTP, or HTTPS receive endpoint is deployed with the public port configuration enabled, the endpoint application is deployed on the public HTTPS port (`8082`), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial#shared-load-balancers[shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint.
+When an AS2, HTTP, or HTTPS receive endpoint is deployed with the public port configuration enabled, the endpoint application is deployed on the public HTTPS port (`8082`), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial.adoc#shared-load-balancers.adoc[shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint.
```
Comment (sathya0) [inline modules/ROOT/pages/deploying-receive-endpoints.adoc]:
```suggestion
When an AS2, HTTP, or HTTPS receive endpoint is deployed with the public port configuration enabled, the endpoint application is deployed on the public HTTPS port (`8082`), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial.adoc[shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint.
```

File: modules/ROOT/pages/deploying-receive-endpoints.adoc [hunk 3]
```diff
@@ -1,55 +1,69 @@
...
 
-You can view this URL by either selecting the receive endpoint from your deployed message flow or clicking the endpoint name from the endpoints list. You can share this URL with the partners that send AS2, HTTP, or HTTPS messages to this endpoint.
+You can view the runtime application name and external URL for a deployed endpoint from the endpoint details page. For example:
+
+image::pm-receive-endpoint-url.png[Example of a runtime application name, internal URL, and external URL]
```
Comment (sathya0) [inline modules/ROOT/pages/deploying-receive-endpoints.adoc]: should this image be under the images folder? i think right now it isnt

File: modules/ROOT/pages/deploying-receive-endpoints.adoc [hunk 4]
```diff
@@ -1,55 +1,69 @@
...
 
-For example, if the name of your DLB domain is `mythical.lb.anypointdns.net` and the runtime application name is `b2b-inbound-as2-zap3`, then your administrator adds a URL mapping rule in the DLB settings to forward incoming requests to the runtime application within your VPC:
+The external URL for an AS2 receive endpoint is `+https://{DLB-domain}/{Input-path}/receive-as2+`.
+
+For example, if the name of your DLB domain is `mythical.lb.anypointdns.net` and the runtime application name is `b2b-inbound-as2-zap3`, then your administrator can add a URL mapping rule in the DLB settings to forward incoming requests to the runtime application within your VPC:
 
 image::url-mapping-rules-as2.png[URL mapping rules for AS2 in the DLB settings]
 
-With these configurations, the external AS2 endpoint URL is `+https://mythical.lb.anypointdns.net/as2/mythical/receive-as2/+`.
+With this configuration the external AS2 endpoint URL is `+https://mythical.lb.anypointdns.net/as2/mythical/receive-as2/+`.
```
Comment (sathya0) [inline modules/ROOT/pages/deploying-receive-endpoints.adoc]:
```suggestion
With this configuration, the external AS2 endpoint URL is `+https://mythical.lb.anypointdns.net/as2/mythical/receive-as2/+`.
```

File: modules/ROOT/pages/deploying-receive-endpoints.adoc (hunk 3 above)
Comment (dhirsch1) [reply modules/ROOT/pages/deploying-receive-endpoints.adoc]: It is there; pm-receive-endpoint.url

File: modules/ROOT/pages/deploying-receive-endpoints.adoc [hunk 5]
```diff
@@ -21,7 +21,7 @@ To deploy an AS2, HTTP, or HTTPS receive endpoint with the private port configur
 * Deploy a message flow that includes the receive endpoint.
 * Set CloudHub to automatically deploy newly created endpoints to a private port, as described in xref:cloudhub-deploy-options.adoc[CloudHub deployment settings].
 
-Endpoints with the private port configuration enabled are deployed to the private HTTPS port `8092`. You can view the runtime application name, internal URL, and external URL for a deployed endpoint from the endpoint details page or the *Receive* section of your message flow.
+Endpoints with the private port configuration enabled are deployed to the private HTTPS port `8092`. You can view the runtime application name, internal URL, and external URL for a deployed endpoint from the endpoint details page, which you can access from the.
```
Comment (sathya0) [inline modules/ROOT/pages/deploying-receive-endpoints.adoc]: unfinished sentence

**Review by sathya0 — APPROVED** [review]

**Pull request merged**